dotnet/IoTPub.cs

```diff
@@ -0,0 +1,4 @@
+IRealtimeChannel channel = ably.Channels.Get(
```

Comment:
can we make this one liner?

Reply:
Not without exceeding the 64 column line length limit.

Reply:
Resolving this conversation so that we can land this work as it's needed for other teams to work on tomorrow.